dbms/src/Dictionaries/ClickHouseDictionarySource.cpp

```diff
@@ -143,4 +159,19 @@ BlockInputStreamPtr ClickHouseDictionarySource::createStreamForSelectiveLoad(con
     return std::make_shared<RemoteBlockInputStream>(pool, query, sample_block, context);
 }
 
+std::string ClickHouseDictionarySource::doInvalidateQuery(const std::string & request) const
+{
+    Block invalidate_sample_block;
```

Comment:
Unused variable in case of `is_local`. Better to create `if (is_local) ... else ...`.